sa/sa.go

```diff
@@ -33,6 +33,7 @@ type certCountFunc func(db db.Selector, domain string, earliest, latest time.Tim
 
 // SQLStorageAuthority defines a Storage Authority
 type SQLStorageAuthority struct {
+	sapb.UnimplementedStorageAuthorityServer
```

Comment:
Same comment here as for the RA -- I think we should only have this show up in one struct, the one which actually currently implements this interface. This can be moved from the wrapper to here when the unwrapping work is complete.